editor/editor_node.cpp

```diff
@@ -5886,8 +5918,10 @@ EditorNode::EditorNode() {
 	editor_layouts->connect("id_pressed", this, "_layout_menu_option");
 	p->add_submenu_item(TTR("Editor Layout"), "Layouts");
 #ifdef OSX_ENABLED
+	p->add_shortcut(ED_SHORTCUT("editor/editor_editor", TTR("Take Screenshot"), KEY_MASK_CMD | KEY_F9), EDITOR_SCREENSHOT);
```

Comment:
You're sure you can't find a worse shortcut identifier? ;)

Reply:
(I mean `editor/editor_editor`).

Reply:
Changed to "editor/take_screenshot".